sbt-yeoman/app/com/tuplejump/sbt/yeoman/Yeoman.scala

```diff
@@ -58,33 +59,30 @@ object Yeoman extends AutoPlugin {
     } else throw new Exception("grunt failed")
   }
 
-  lazy val withTemplates = {
-    Seq(sourceDirectories in TwirlKeys.compileTemplates in Compile ++= Seq(yeomanDirectory.value / "dist"),
-      yeomanExcludes <<= (yeomanDirectory) (yd => Seq(
-        yd + "/dist/components/",
-        yd + "/dist/images/",
-        yd + "/dist/styles/"
-      )),
-      excludeFilter in unmanagedSources <<=
-        (excludeFilter in unmanagedSources, yeomanExcludes) {
-          (currentFilter: FileFilter, ye) =>
-            currentFilter || new FileFilter {
-              def accept(pathname: File): Boolean = {
-                (true /: ye.map(s => pathname.getAbsolutePath.startsWith(s))) (_ && _)
-              }
-            }
-        },
-      (compile in Compile) <<= (compile in Compile) dependsOn gruntDist
-    )
-  }
+//  lazy val withTemplates = {
```

Comment:
I couldn't quite figure out how to upgrade this section. I didn't see any references to `withTemplates` so I commented it out for my own local purposes. Can we safely remove it?